corsheaders/checks.py

```diff
@@ -0,0 +1,88 @@
+import collections
+import numbers
+
+from django.core import checks
+from .conf import corsheaders_settings
+
+@checks.register
+def check_settings(app_configs, **kwargs):
+
+    errors = []
+    if not isinstance(corsheaders_settings.CORS_ALLOW_HEADERS, collections.Sequence):
+        errors.append(
+            checks.Warning(
```

Comment:
I think we should use `checks.Error` since that will cause a failure - warnings just get printed out, and typically ignored.